src/olympia/reviewers/serializers.py

```diff
@@ -411,9 +412,12 @@ def _trim_entry(self, entry):
                 ) if key in entry}
 
     def get_validation_url_json(self, obj):
-        return absolutify(reverse('reviewers.json_file_validation', args=[
-            obj.addon.pk, obj.current_file.id
-        ]))
+        return absolutify(drf_reverse(
+            'reviewers-addon-json-file-validation',
+            request=self._context.get('request'),
```

Comment:
We have some code using `self._context` but we should change it (no idea why it was done that way), `self.context` is the supported way of accessing the context in a serializer.